src/hotspot/share/opto/node.hpp

```diff
@@ -1452,6 +1458,8 @@ class DUIterator_Last : private DUIterator_Fast {
   DUIterator_Last() { }
   // initialize to garbage
 
+  DUIterator_Last (const DUIterator_Last& that) = default;
```

Comment:
Nit - remove space before parameter list.

Reply:
Thanks for point it out. Will remove.